src/backend/marsha/core/management/commands/send_reminders.py

```diff
@@ -0,0 +1,155 @@
+    REMINDER_ERROR,
+    REMINDER_IS_STARTED,
+    REMINDER_KEY_REGISTER_BEFORE_S,
+    REMINDER_KEY_STARTS_IN_S,
+    REMINDERS_STEP,
+    RUNNING,
+)
+from marsha.core.models import LiveRegistration
+
+
+logger = getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """Send reminders for scheduled webinar."""
+
+    help = "Send reminders for scheduled webinar."
+
+    def query_to_update_step(self, liveregistration, step):
+        """scripts could be called simultaneously, we update reminders field
+        using strict where clauses, the same ones used in the select query."""
+
+        if liveregistration.reminders:
+            liveregistration.reminders.append(step)
+        else:
+            liveregistration.reminders = [step]
+
+        # to send mails if still appropriate
+        return (
+            LiveRegistration.objects.filter(
+                id=liveregistration.id, should_send_reminders=True
+            )
+            .exclude(reminders__overlap=[step, REMINDER_IS_STARTED, REMINDER_ERROR])
+            .update(reminders=liveregistration.reminders)
+        )
+
+    def send_reminders_and_update_liveregistrations_step(
+        self, liveregistrations, step, template="reminder"
+    ):
+        """Send email with template and update reminders field."""
+        for liveregistration in liveregistrations:
+            """For each liveregistration we send the email"""
+
+            # if record was updated
+            if self.query_to_update_step(liveregistration, step) == 1:
+
+                self.stdout.write(
+                    f"Sending email for liveregistration {liveregistration.id} "
+                    f"for video {liveregistration.video.id} step {step}"
+                )
+
+                # send email with the appropriate template and object
+                vars = (
+                    {
+                        "reminder_timer_text": _(f"reminder_timer_text_{step}"),
+                        "reminder_timer_title": _(f"reminder_timer_title_{step}"),
+                    }
+                    if template == "reminder"
+                    else {}
+                )
+
+                msg_plain = render_to_string(f"core/mail/text/{template}.txt", vars)
+                msg_html = render_to_string(f"core/mail/html/{template}.html", vars)
+
+                try:
+                    send_mail(
+                        _(f"object_{template}_{step}"),
+                        msg_plain,
+                        settings.EMAIL_FROM,
+                        [liveregistration.email],
+                        html_message=msg_html,
+                        fail_silently=False,
+                    )
+                except smtplib.SMTPException as exception:
+                    # send error to sentry and print it
+                    liveregistration.update_reminders(REMINDER_ERROR)
+                    self.stdout.write(f"Mail failed {liveregistration.email} ")
+                    capture_message(
```

Comment:
You should capture the exception here, not a message

Reply:
I don't throw the exception anymore and I mock it how could I now, I was catching it before when I used a raise, now none is thrown

Reply:
ah Ok i get it !

Reply:
no actually, it's not enough {str(exception)} wdym by I should capture the exception

Reply:
yes capture_exception, I got it...

Reply:
I mean you should use the function `capture_exception` available in the sentry sdk: https://docs.sentry.io/platforms/python/usage/
`capture_exception(exception)`